src/node_http2.cc

```diff
@@ -426,8 +426,13 @@ Http2Priority::Http2Priority(Environment* env,
                              Local<Value> weight,
                              Local<Value> exclusive) {
   Local<Context> context = env->context();
-  int32_t parent_ = parent->Int32Value(context).ToChecked();
-  int32_t weight_ = weight->Int32Value(context).ToChecked();
+  int32_t parent_;
+  int32_t weight_;
+  if (!parent->Int32Value(context).To(&parent_) ||
+      !weight->Int32Value(context).To(&weight_)) {
+    nghttp2_priority_spec_init(this, 0, 0, 0);
```

Comment:
Should we still call this in the error path? Could we add a comment explaining why? My instinct would be we should return and let V8 throw an error

Reply:
We are returning to let it throw an error. Calling this should be perfectly safe.